anthos-bm-gcp-terraform/docs/manuallb_install.md

```diff
@@ -0,0 +1,299 @@
+## Install with Manual Loadbalancer
+
+This guide is an extension to the [all-in-one install guide](one_click_install.md).
+The `all-in-one install` is a single run installation which automatically
+triggers the complete **Anthos on bare metal** installation when the Terraform
+script is run. The default installation sets up Anthos on bare metal to run
+using a [bundled loadbalancer](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/installing/bundled-lb).
+This means that the loadbalancer and the cluster services are reachable only
+from inside one of the GCE VMs _(i.e. Admin workstation, Controlplane nodes, Worker nodes)_
+created during the installation process.
+
+While, this setup closely represents how **Anthos on bare metal (ABM)** will be
```

Comment:
nit: ' While this setup'

Reply:
Fixed in: 81e810a86a4cc635b10cc2de13a410a55ef56d21